src/components/List/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useLocation, useHistory } from 'react-router-dom';
+
+const StyledTD = styled.td`
+  cursor: pointer;
+`;
+
+const List = ({ columnOrder, invisibleColumns, items, primaryKeyColumn }) => {
+  const location = useLocation();
+  const history = useHistory();
+
+  if (!items.length) {
+    return null;
+  }
+
+  const filterVisibleColumns = colHeader =>
+    invisibleColumns.includes(colHeader) === false;
+
+  const colHeaders =
+    columnOrder || Object.keys(items[0]).filter(filterVisibleColumns);
+
+  const onRowClick = e => {
+    const { dataset } = e.currentTarget;
+    const { itemId } = dataset;
+
+    if (itemId) {
+      history.push(`${location.pathname}/${itemId}`);
+    }
+  };
+
+  return (
+    <table cellPadding="0" cellSpacing="0" width="100%">
+      <thead>
+        <tr>
+          {colHeaders.map(colHeader => (
+            <th key={colHeader}>{colHeader}</th>
+          ))}
+        </tr>
+      </thead>
+
+      <tbody>
+        {items.map((row, rowIndex) => (
+          <tr
+            key={JSON.stringify(items[rowIndex])}
+            data-item-id={primaryKeyColumn && items[rowIndex][primaryKeyColumn]}
```

Comment:
is dat een typo?

Reply:
Which character on this line are you referring to?

Reply:
You stalking me, @rkedde?

Reply:
data-typeid?

blijkbaar niet